corehq/apps/reports/standard/monitoring.py

```diff
@@ -152,6 +148,33 @@ class CaseActivityReport(WorkerMonitoringCaseReportTableBase):
     description = ugettext_noop("Followup rates on active cases.")
     is_cacheable = True
 
+    @property
+    def landmark_columns(self):
+        return [
+            (_("# Updated or Closed"),
+             _("The number of cases that have been created, updated, or closed "
+               "between {} days ago and today.")),
+            (_("# Active"),
+             _("The number of open cases created or updated in the last {} days.")),
+            (_("# Closed"),
+             _("The number of cases that have been closed between {} days ago and "
+               "today.")),
+            (_("Proportion"),
+             _("The percentage of all recently active cases that were created, "
+               "updated or closed in the last {} days.")),
+            # "recently active" means "touched in the last 120 days"
+        ]
+
+    @property
+    def totals_columns(self):
+        return [
+            (_("# Active Cases"),
+             _("Number of open cases modified in the last {} days")),
+            (_("# Inactive Cases"),
+             _("Number of cases that are open but haven't been touched in the "
+               "last %s days")),
```

Comment:
Thanks, updated.  I could've sworn `"something".format(var)` would fail hard if there is no `{}` in the string.